database/schema.sql

```diff
@@ -0,0 +1,6 @@
+DROP TABLE IF EXISTS twitbot;
+
+CREATE TABLE IF NOT EXISTS twitbot (
+  id SERIAL PRIMARY KEY,
+  tweets TEXT
```

Comment:
I would make the column `tweets` non nullable (ie you have to pass it in a value).